dm/proto/dmworker.proto

```diff
@@ -166,6 +166,8 @@ message SyncStatus {
     int64 totalRows = 15;
     int64 totalRps = 16;
     int64 recentRps = 17;
+    uint64 iOTotalBytes = 18;
```

Comment:
```suggestion
    uint64 ioTotalBytes = 18;
```

Reply:
and please explain what `ioTotalBytes` used for in comments

Reply:
fix in https://github.com/pingcap/tiflow/pull/11742/commits/a4d5ba2f44fd7da8ea5a8a5e309c315d2298e5b1